examples/text_classification/iterable_train.py

```diff
@@ -148,7 +147,7 @@ def get_csv_iterator(data_path, ngrams, vocab, start=0, num_lines=None):
     """
     def iterator(start, num_lines):
         tokenizer = get_tokenizer("basic_english")
-        with io.open(data_path, encoding="utf8") as f:
+        with open(data_path, encoding="utf8") as f:
```

Comment:
Is `io.open` function, rather than built-in `open` function, recommended for Python3?

Reply:
In Python 3, `io.open` is an alias for builtin `open`:

https://docs.python.org/3/library/io.html#io.open